### ISSUE TITLE
Feature request : Keybind to show/hide weapon/spell range

### ISSUE THREAD

**Issue by uniwars** [Aedif/tactical-grid#34]
Hi, would that be possible to add a setting and a keybind to show/hide weapon/spell range when hovering it ? Like the keybind to see distance/cover.

**Comment by Aedif** [comment]
As in you want to be able to activate range highlight on command? Instead of it always activating when hovering over?

**Comment by uniwars** [comment]
Yes, either by having a keybind to toggle it on/off and use the same behavior when it's on, or having a keybind to press in addition to hovering. The idea is to let each user have a setting as they like, turned off, turned on, or turned on on demand

**Comment by Aedif** [comment]
New keybinding added to toggle range highlighting on client side.
https://github.com/Aedif/tactical-grid/releases/tag/1.27.0
